src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -18,7 +18,8 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(ReadOnlyPerson person) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword)) ? true
+        : keywords.stream().anyMatch(keyword -> person.getName().fullName.toLowerCase().contains(keyword.toLowerCase()));
```

Comment:
Wrong indentation? [Our coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout) requires 4 spaces (instead of tabs) for normal indentation and an indentation of 8 spaces for line wrapping (i.e. when breaking a long line into multiple lines).